src/__tests__/cacao-siwe.test.ts

```diff
@@ -1,169 +1,110 @@
 import { Wallet } from '@ethersproject/wallet'
 import { Cacao, CacaoBlock } from '../cacao.js'
 import { SiweMessage } from '../siwx/siwe.js'
+import { DateTime } from 'luxon'
+
+const ISSUED_AT = DateTime.fromISO('2021-10-14T07:18:41Z').toUTC()
```

Comment:
Why not use Date?

Reply:
Because it's clunky. We are adding and removing time to this time stamp in the tests. Using vanilla Date makes the code barely readable.